### ISSUE TITLE
[FeatureReq] negpip

### ISSUE THREAD

**Issue by za-wa-n-go** [shiimizu/ComfyUI_smZNodes#12]
It would be great if negpip would be available in ClipTextEncode++ as it seems very useful.

https://github.com/laksjdjf/cd-tuner_negpip-ComfyUI

**Comment by toyxyz** [comment]
Yes! This extension is very useful. It would be very cool if it was available for ComfyUI.

**Comment by toyxyz** [comment]
Here's a ComfyUI Negpip custom node, but it throws an error when used with smZNodes.
- https://github.com/laksjdjf/cd-tuner_negpip-ComfyUI
-
![image](https://github.com/shiimizu/ComfyUI_smZNodes/assets/8006000/e3a82307-8ae2-42ba-9d63-017a609b428c)
